core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -218,7 +218,7 @@ var ExplorationEditorMainTab = function() {
       'Save',
     ];
     for (const HEADING of tutorialTabHeadings) {
-    // Use: await tutorialTabHeadings.forEach(async function(heading) {
```

Comment:
Does this raise a lint error? It shouldn't because we shouldn't care about comments

Reply:
It doesn't.